blocks/autocompleters/index.js

```diff
@@ -125,3 +132,56 @@ export function userAutocompleter() {
 		onSelect,
 	};
 }
+
+export function hashtagAutocompleter() {
+	const c = wp.api.collections;
```

Comment:
Destructuring can help here to avoid using abbreviated `c` const:

```js
const { Categories, Pages, Posts, Tags } = wp.api.collections;
```

Reply:
Good idea